packages/affiliates/apps/DevMiningRewards.js

```diff
@@ -47,7 +53,19 @@ const App = env => async params => {
     rewards.utils.validateEmpInput(empInput);
     // convert to standard eth checksum address otherwise lookups through BQ or web3 will fail
     // Allow for non standard payout address at empInput[1] since this has no impact on processing.
-    return [rewards.utils.toChecksumAddress(empInput[0]), empInput[1]];
+    let [empAddress, payoutAddress, empVersion] = empInput;
+
+    // we want to make sure these addresses are standarized since we do many lookups internally
+    empAddress = rewards.utils.toChecksumAddress(empAddress);
+
+    // this converts a version number from the config into an abi which gets passed into the calculator
+    // so each emp contract will have the correct abi passed along with it.
+    let empAbi;
+    if (empVersion) {
+      empAbi = getAbi("ExpiringMultiParty", empVersion);
+    }
```

Comment:
i can consider changing this.  the only reason i didnt was because this would change the behavior of the lower level functions and more tests would need to be updated.